index.html

```diff
@@ -9,5 +9,6 @@
   <body>
     <div id="app"></div>
     <script type="module" src="/src/main.ts"></script>
+    <link href="https://cdn.jsdelivr.net/npm/@mdi/font@4.x/css/materialdesignicons.min.css" rel="stylesheet">
```

Comment:
Linking resources from the web is slower and less reliable than hosting it ourselves, consider putting the icon font you want into our repo?

Reply:
I think we can make seperate user story towards end of project once we have all the file icons we need to do this. FOr now this is low priority in my eyes.